tests/rptest/services/redpanda.py

```diff
@@ -594,9 +594,12 @@ def set_extra_node_conf(self, node, conf):
         assert node in self.nodes
         self._extra_node_conf[node] = conf
 
-    def set_security_settings(self, settings):
+    def set_security_settings(self,
```

Comment:
It took me a while to figure out why this change is required. It's so that certificates are not **re**-created, right?

Reply:
Correct.

Reply:
I'll put that in the commit description and comments

Reply:
I added that into the commit for clarification.